modules/overview-of-openshift-gitops.adoc

```diff
@@ -0,0 +1,32 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/working-with-openshift-gitops.adoc
+
+[id="overview-of-openshift-gitops_{context}"]
+= Overview of OpenShift GitOps
+
+Red Hat OpenShift GitOps is a set of practices that use Git pull requests to manage infrastructure and application configurations. The Git repository in Red Hat OpenShift GitOps is the only source of truth for system and application configuration. The repository contains the entire state of the system so that the trail of changes to the system state are visible and auditable.
+
+Use Red Hat OpenShift GitOps tooling to create repeatable and predictable processes for managing and recreating OpenShift Container Platform clusters and applications. By using Red Hat OpenShift GitOps, you resolve the issues of infrastructure and application configuration sprawl.
```

Comment:
s/ Use Red Hat OpenShift GitOps tooling/You can use GitOps tooling
s/Red Hat OpenShift GitOps/ GitOps througout this paragraph.